Sample/Section.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Section.swift
+//  Sample
+//
+//  Created by Zev Eisenberg on 9/10/20.
+//  Copyright © 2020 The New York Times. All rights reserved.
+//
+
+enum Section: CaseIterable {
```

Comment:
We added this to standardize the section logic between the two demos. Previously, the Texture example was making a peculiar use of modular arithmetic to make a quasi-FizzBuzz, pseudorandom distribution of cells that was probably slightly more realistic, but also made testing very confusing.